App/NativeModules/Textile/Model.ts

```diff
@@ -106,10 +106,12 @@ export interface CafeSession {
 export interface ContactInfo {
   readonly id: string
   readonly address: string
-  readonly username: string
-  readonly inboxes: ReadonlyArray<Cafe>
-  readonly added: string
-  readonly thread_ids: ReadonlyArray<string>
+  readonly username?: string
+  readonly avatar?: string
+  readonly inboxes?: ReadonlyArray<Cafe>
+  readonly created: string
+  readonly updated: string
+  readonly thread_ids?: ReadonlyArray<string>
```

Comment:
This `ContactInfo` model needed updating. Using it for both `addContact` and the return value for `contact()` and `contacts()` since the optional `thread_ids` is the only thing that doesn't exist on the go layer's `Contact` model.